app/assets/images/payment_icons/spraypay.svg

```diff
@@ -0,0 +1 @@
fill="#0057a5"/><g transform="matrix(.2234 0 0 .19436 1.324 5.761)" id="pi-g4016"><path class="st0" d="M74.04 18.19s2.35 2.67 5.2 2.67c1.14 0 2.29-.41 2.29-1.68 0-2.51-10.28-2.79-10.28-9.9 0-4.09 3.59-6.82 8.09-6.82 3.01 0 7.24 1.46 7.24 4.92v2.48H81.5v-1c0-.82-1.17-1.33-2.22-1.33-1.14 0-2.32.48-2.32 1.52 0 2.76 10.28 2.38 10.28 9.84 0 3.78-2.95 7.01-8.06 7.01-5.33 0-8.28-3.74-8.28-3.74z" id="pi-path4000" fill="#0057a5"/><path class="st0" d="M90.85 14.48c0-.44-.25-.7-.7-.7h-1.21v-4.6h4.73c1.59 0 2.13.73 2.13 1.43v.29h.06s1.43-2.09 4.89-2.09c4.44 0 7.36 3.49 7.36 8.57 0 5.17-3.27 8.54-7.62 8.54-2.67 0-4.16-1.4-4.16-1.4h-.06s.06.57.06 1.4v5.97h-5.49V14.48zm8.54 6.88c1.71 0 3.2-1.4 3.2-3.9 0-2.44-1.3-4.03-3.2-4.03-1.65 0-3.21 1.21-3.21 4.03 0 1.97 1.05 3.9 3.21 3.9z" id="pi-path4002" fill="#0057a5"/><path class="st0" d="M111.41 14.48c0-.44-.25-.7-.7-.7h-1.21v-4.6h4.86c1.49 0 2.32.6 2.32 2.09v.86c0 .51-.06.82-.06.82h.06c.7-2.22 2.76-4.06 5.2-4.06.32 0 .63.06.63.06v5.39s-.44-.1-1.11-.1c-1.3 0-3.43.41-4.22 2.86-.19.6-.29 1.33-.29 2.22v6.19h-5.49V14.48z" id="pi-path4004" fill="#0057a5"/><path class="st0" d="M132.73 15.24h.22v-.19c0-1.55-.73-2.13-2.09-2.13-.7 0-1.43.16-1.43.79v.73h-5.08v-1.81c0-3.65 4.63-3.84 6.51-3.84 5.71 0 7.58 2.92 7.58 6.66v4.76c0 .44.25.7.7.7h1.21v4.6h-4.76c-1.59 0-2.16-.98-2.16-1.71v-.29h-.06s-1.14 2.38-4.57 2.38c-3.05 0-5.55-2.03-5.55-5.27 0-4.84 6.95-5.38 9.48-5.38zm-2.41 6.47c1.43 0 2.7-1.55 2.7-2.89v-.25h-.51c-1.74 0-3.74.48-3.74 1.78 0 .76.47 1.36 1.55 1.36z" id="pi-path4006" fill="#0057a5"/><path class="st0" d="M144.73 27.62c.89 0 1.9-.25 2.44-1.62l.19-.48-5.17-11.23c-.19-.41-.32-.51-.7-.51h-.6v-4.6h4c.92 0 1.59.48 1.9 1.33l2.41 6.6c.32.89.6 2.54.6 2.54h.06s.29-1.65.54-2.54l1.81-6.6c.22-.86 1.08-1.33 1.94-1.33h4.09v4.6h-.6c-.38 0-.6.1-.76.51l-4.95 12.98c-1.4 3.62-4.22 4.92-6.82 4.92-2.82 0-4.6-1.74-4.6-1.74l1.97-3.87c0-.01.95 1.04 2.25 1.04z" id="pi-path4008" fill="#0057a5"/><path class="st1" d="M72.96 37.89H70.9v-4.76h11.01c4.41 0 7.39 3.24 7.39 7.81 0 4.57-2.98 7.87-7.39 7.87h-3.27v7.01h-5.68zm7.81 6.16c1.84 0 2.73-1.3 2.73-3.11 0-1.81-.89-3.05-2.63-3.05h-2.22v6.16z" id="pi-path4010" fill="#00853e"/><path class="st1" d="M99.58 45.54h.22v-.19c0-1.55-.73-2.13-2.09-2.13-.7 0-1.43.16-1.43.79v.73H91.2v-1.81c0-3.65 4.63-3.84 6.51-3.84 5.71 0 7.58 2.92 7.58 6.66v4.76c0 .44.25.7.7.7h1.21v4.6h-4.76c-1.59 0-2.16-.98-2.16-1.71v-.29h-.06s-1.14 2.38-4.57 2.38c-3.05 0-5.55-2.03-5.55-5.27-.01-4.84 6.94-5.38 9.48-5.38zm-2.41 6.47c1.43 0 2.7-1.56 2.7-2.89v-.25h-.51c-1.74 0-3.74.48-3.74 1.78-.01.76.47 1.36 1.55 1.36z" id="pi-path4012" fill="#00853e"/><path class="st1" d="M111.57 57.91c.89 0 1.9-.25 2.44-1.62l.19-.48-5.17-11.23c-.19-.41-.32-.51-.7-.51h-.6v-4.6h4c.92 0 1.59.48 1.9 1.33l2.41 6.6c.32.89.6 2.54.6 2.54h.06s.29-1.65.54-2.54l1.81-6.6c.22-.86 1.08-1.33 1.94-1.33h4.09v4.6h-.6c-.38 0-.6.1-.76.51l-4.95 12.98c-1.4 3.62-4.22 4.92-6.82 4.92-2.82 0-4.6-1.74-4.6-1.74l1.97-3.87s.95 1.04 2.25 1.04z" id="pi-path4014" fill="#00853e"/></g></svg>
```

Comment:
There are `id` attributes that are not needed. Can you remove them except the `<title>` tag?

[![alt](https://screenshot.click/16-39-l2ajs-lv1fy.png)](https://screenshot.click/16-39-l2ajs-lv1fy.png)